servers/visual/visual_server_canvas.cpp

```diff
@@ -2056,6 +2196,81 @@ void VisualServerCanvas::_print_tree_down(int p_child_id, int p_depth, const Ite
 
 #endif
 
+void VisualServerCanvas::tick() {
+	if (_interpolation_data.interpolation_enabled) {
+		update_interpolation_tick(true);
+	}
+}
+
+void VisualServerCanvas::update_interpolation_tick(bool p_process) {
+#define GODOT_UPDATE_INTERPOLATION_TICK(LIST_PREV, LIST_CURR, TYPE, OWNER_LIST)              \
+	/* Detect any that were on the previous transform list that are no longer active. */     \
+	for (unsigned int n = 0; n < _interpolation_data.LIST_PREV->size(); n++) {               \
+		const RID &rid = (*_interpolation_data.LIST_PREV)[n];                                \
+		TYPE *item = OWNER_LIST.getornull(rid);                                              \
+		/* no longer active? (either the instance deleted or no longer being transformed) */ \
+		if (item && !item->on_interpolate_transform_list) {                                  \
+			item->xform_prev = item->xform_curr;                                             \
+		}                                                                                    \
+	}                                                                                        \
+	/* and now for any in the transform list (being actively interpolated), */               \
+	/* keep the previous transform value up to date and ready for next tick */               \
```

Comment:
```suggestion
	// Detect any that were on the previous transform list that are no longer active.        \
	for (unsigned int n = 0; n < _interpolation_data.LIST_PREV->size(); n++) {               \
		const RID &rid = (*_interpolation_data.LIST_PREV)[n];                                \
		TYPE *item = OWNER_LIST.getornull(rid);                                              \
		/* no longer active? (either the instance deleted or no longer being transformed) */ \
		if (item && !item->on_interpolate_transform_list) {                                  \
			item->xform_prev = item->xform_curr;                                             \
		}                                                                                    \
	}                                                                                        \
	// And now for any in the transform list (being actively interpolated),                  \
	// keep the previous transform value up to date and ready for next tick.                 \
```

Reply:
Don't know how that crept in! Must have been one of my editors changing the comment style automatically. Will fix.